### ISSUE TITLE
Access to path in onMoveNode

### ISSUE THREAD

**Issue by rohitgupta3** [fritz-c/react-sortable-tree#25]
Is there any way to access the path of a node in the onMoveNode callback?  My specific use case is when a node is moved I want to know its new parent and not sure of the easiest way to do this with the existing API.  I could traverse the treeData manually to find the node's new parent but I was hoping with the existing API this can be done easily - just having trouble figuring out how.

Appreciate any help and thanks so much for this excellent component.

EDIT: it is very easy to just traverse the tree and find the parent, so I would just remove this issue - I can't seem to figure out how though...Perhaps only @fritz-c can do so.  Sorry about this.

**Comment by fritz-c** [comment]
I thought about adding this functionality, but held back as I didn't know what the usefulness of that information would be. If you don't mind, could you explain in a bit more detail why you need to know the parent?
In any case, I'll close the issue, so no worries.

**Comment by rohitgupta3** [comment]
Sure - though this issue is not an issue anymore, if it's interesting / relevant I have hierarchical product data.  So bananas and apples could be children of fruits, celery and cucumbers are children of vegetables.  If, in the database, product A is incorrectly the child of product X and should instead be the child of product Y, then after an admin drags and drops A under Y, I need to tell the database that the parent of A should be Y.  Is there an easier way of doing this than finding the parent (which was pretty straightforward anyway)?

**Comment by fritz-c** [comment]
Ah, whenever the node position is changed, it gets saved immediately on the server. That makes sense.

This seemed like a common enough issue that I just added this functionality in `v0.1.7`. Let me know if it works for you!

Thanks for making the issue!

**Comment by rohitgupta3** [comment]
Will try to check it out soon and let you know!

Separately, as a random heads-up, when I randomly did a fresh npm install and got `v0.1.7` with  dependency `react-virtualized v8.6.1`, the list broke (I would get a blank screen as I scrolled down since the DOM did not update the list items).  Think the issue is react-virtualized because `v0.1.7` with `react-virtualized v8.5.3` was ok.

Don't quote me on any of this though as I did it quickly then reverted, but may be useful info if you find yourself in a situation where you find scrolling down is not working.
